_posts/2023-05-12-sprint-attendees.md

```diff
@@ -0,0 +1,39 @@
+
+Over time, these core developers realized sprints also provided them an opportunity to help find new potential contributors. And that’s where YOU come in.
+
+## Why Attend a Sprint at DJEU 2023?
+Sprints give you an opportunity to work shoulder-to-shoulder with experts in the community. In addition, there are practical and personal benefits to attending the sprints.
+
+From a practical perspective, if your organization makes heavy use of Django or a related package, getting involved in its maintenance will give you a better understanding of that package, as well as it’s strengths & limitations. On the personal side, most people won’t usually work on a project that’s as widely used as Django or some of the other related packages. Getting involved in the maintenance of one of these packages can expose you to engineering practices you like wouldn’t see otherwise. This can help you become a better engineer while allowing you to contribute to the community that benefits you and your company.
+
+## What should you know if you want to sprint?
+Git and Github basics
+Most FLOSS projects these days use Git for version control (the software used for tracking changes to software). This makes it easier for groups of people to work on the same software, together. If you’re new to Git, I recommend the first three chapters of Pro Git, which is freely available to read online or to download. Those chapters will get you started and give you about 80% of the Git you’ll use most often. Chapters five and six, on Distributed Git and Github respectively, are also very helpful if you’re new to working with a group on software.
+
+If you’re new to Git or working on software in teams, a basic knowledge of Git fundamentals is important to helping maintainers. Keep in mind that one of the main focuses of contributing to a FLOSS project is to help the people who maintain the software we all use. So, while the sprints aren’t the best place to learn Git, they are a great opportunity to practice or improve your Git skills, especially in situations you may not run into normally.
+
+In the spirit of helping our maintainers, try to focus on learning to make “good” commits. More specifically, you want to make atomic commits, in which each commit is focused around a specific activity or code change, and clear commit messages, meaning that the changes in a commit should be clear from its commit message. This is important because commits you make will be relied upon by current or future contributors & maintainers. Making “good” commits and messages is both an art and a science, so practice or experience helps. You should also have a GitHub account. Git and GitHub, while related, are two different things, so getting Git setup on your computer won’t give you a GitHub account, just as creating a GitHub account doesn’t mean you’ll have Git installed on your computer.
```

Comment:
```suggestion
In the spirit of helping our maintainers, try to focus on learning how to make “good” commits. More specifically, you want to make atomic commits, in which each commit is focused around a specific activity or code change, with clear commit messages, meaning that the changes in a commit should be clear from its commit message. This is important because the commits you make will be relied upon by current or future contributors and maintainers. Making “good” commits and messages is both an art and a science, so practice and experience helps. You should also have a GitHub account. Git and GitHub, while related, are two different things, so getting Git setup on your computer won’t give you a GitHub account, just as creating a GitHub account doesn’t mean you’ll have Git installed on your computer.
```